talk-ink/talk-ink
# Web 100 inbox improvment

- Open and close thread should sort by newest
- Open and close should be realtime
- Add comment before open closed thread
- Add field latestActionAt
- Delete should not permanent use draft instead
- Realtime delete thread
- Change date from 2 hours to 2.00PM on thread item
